docs/languages/java.md

```diff
@@ -0,0 +1,105 @@
+---
+Order:
+Area: languages
+TOCTitle: Java
+ContentId:
+PageTitle: Java in VS Code
+DateApproved:
+MetaDescription: Learn about Visual Studio Code editor features (code completion, debugging, snippets, linting) for Java.
+---
+# Java in VS Code
+
+Java is fully supported in Visual Studio Code through [extensions](/docs/editor/extension-gallery.md). Popular extensions in the [Marketplace](https://marketplace.visualstudio.com) provide project support, code completion, linting, debugging, code formatting, snippets, and more.
+
+> [Download VS Code](/download) - If you haven't downloaded VS Code yet, quickly install for your platform (Windows, Mac, Linux).
+
+## Install Java Extensions
+
+VS Code is a fast editor and ships with only the basic features. Add Java language support to VS Code by installing the popular Python extensions.
+
+1. Select an extension.
+2. Install the extension by typing `ext install` into the Command Palette `kb(workbench.action.showCommands)`.
+
+<div class="marketplace-extensions-java"></div>
```

Comment:
This query doesn't exist. Do we want to have a curated list, run a query or leave out these tiles all together.

Reply:
Seems like those kind of query is commonly used in other language's pages, such as php and python. I think we don't need a curated list here, just leave out all Java related tiles.